### PR TITLE
SSCS-2690[SA]:Implement logging in app insights on tribunals api

### PR DESCRIPTION
**Pull request by savirneni**
Implement logging in app insights on tribunals api

https://tools.hmcts.net/jira/browse/SSCS-2690

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -104,6 +104,7 @@ dependencies {
 
     compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.4.2'
```
Comment (iguannago) [inline build.gradle]: move to 1.6.1 as well.

File: build.gradle [hunk 2]
```diff
@@ -104,6 +104,7 @@ dependencies {
...
     compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '1.4.2'
```
Comment (iguannago) [inline build.gradle]: move to 1.6.1

File: build.gradle (hunk 1 above)
Comment (savirneni) [reply build.gradle]: Updated to correct version.

File: build.gradle (hunk 2 above)
Comment (savirneni) [reply build.gradle]: Updated to correct version.

**Pull request merged**